independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/ServerSerialisers.java

```diff
@@ -529,7 +529,7 @@ public static void encodeResponseHeaders(ResteasyReactiveRequestContext requestC
                         vertxResponse.addResponseHeader(header, (CharSequence) HeaderUtil.headerToString(o));
                     }
                 }
-                if (header.equals("Transfer-Encoding")) { // using both headers together is not allowed
+                if (header.equalsIgnoreCase("Transfer-Encoding")) { // using both headers together is not allowed
```

Comment:
I agree that it's probably better to do it this way.

I'm surprised though that we are not consistent in the case of the headers. It's Quarkus setting the `Transfer-Encoding` lowercase, you are not adding it manually?

Because if we're not consistent, that also makes the line just below (`vertxResponse.removeResponseHeader("Content-Length");`) a bit brittle.

@geoand WDYT?

Reply:
Weird, I'll have to have a closer look

Reply:
I want to see how this header gets included...

Reply:
I can't reproduce this